Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -32,39 +32,36 @@ public NoOpRunner(IConnectionManager connectionManager)
 
             /* SHAPE FACTORY DESIGN PATTERN */
             ShapeFactory shapeFactory = new ShapeFactory();
-            BaseShape shape1 = shapeFactory.GetShape(Shape.Square, 5, 5);
-            BaseShape shape2 = shapeFactory.GetShape(Shape.Circle, 10, 5);
-            BaseShape shape3 = shapeFactory.GetShape(Shape.Rectangle, 15, 5);
+            BaseShape shape1 = shapeFactory.GetShape(Shape.HealthCrystal, 1, 15);
+            BaseShape shape2 = shapeFactory.GetShape(Shape.DamageCrystal, 5, 15);
+            BaseShape shape3 = shapeFactory.GetShape(Shape.Saw, 8, 15);
             GamePlatforms.AddShape(shape1);
             GamePlatforms.AddShape(shape2);
             GamePlatforms.AddShape(shape3);
 
             /* ABSTRACT SHAPE FACTORY DESIGN PATTERN */
             AbstractFactory abstractShapeFactory = FactoryProducer.GetFactory(true);
```

Comment:
**[Nitpick]** In cases like these, use named arguments (`passable: true)` for better code readability

Reply:
Done